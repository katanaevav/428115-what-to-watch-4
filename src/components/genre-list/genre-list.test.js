import React from "react";
import renderer from "react-test-renderer";
import GenreList from "./genre-list.jsx";

const CURRENT_GENRE = `Fantasy`;

const GENRES = [`Drama`, `Fantasy`];

it(`Render filter genres buttons`, () => {
  const tree = renderer.create(
      <GenreList
        currentGenre = {CURRENT_GENRE}
        genres = {GENRES}
        onGenreClick = {() => {}}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
