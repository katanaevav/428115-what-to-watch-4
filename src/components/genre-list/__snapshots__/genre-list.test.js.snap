// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render filter genres buttons 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      data-key="Drama"
      href="#"
      onClick={[Function]}
    >
      Drama
    </a>
  </li>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      data-key="Fantasy"
      href="#"
      onClick={[Function]}
    >
      Fantasy
    </a>
  </li>
</ul>
`;
